src/api/goods.ts

```diff
@@ -10,10 +10,14 @@ export function getAll(): Promise<Good[]> {
 
 export const get5First = () => {
   return getAll()
-    .then(goods => goods); // sort and get the first 5
+    .then(goods => goods)
+    .then(goods => goods
+      .sort((a, b) => a.name.localeCompare(b.name))
+      .slice(0, 5));
```

Comment:
You don't need this additional `then`
```suggestion
    .then(goods => goods
      .sort((a, b) => a.name.localeCompare(b.name))
      .slice(0, 5));
```